buku.py

```diff
@@ -2545,31 +2659,52 @@ def main():
 
                     pos -= 1
         else:
-            for idx in args.update:
-                if is_int(idx):
-                    bdb.update_rec(int(idx), url_in, title_in, tags, desc_in,
-                                   args.immutable, args.threads)
-                elif '-' in idx and is_int(idx.split('-')[0]) \
-                        and is_int(idx.split('-')[1]):
-                    lower = int(idx.split('-')[0])
-                    upper = int(idx.split('-')[1])
-                    if lower > upper:
-                        lower, upper = upper, lower
+            if args.editor is not None: # check for editor mode
+                # currently allow only editing of one url
+                if len(args.update) != 1 or not is_int(args.update[0]):
```

Comment:
when i use nargs='?', args.write is both None for when i specify -w without argument and when I don't specify -w.